.github/workflows/llvm-binaries.yml

```diff
@@ -365,6 +351,20 @@ jobs:
         run: |
           pwsh -File ${{ github.workspace }}/SourceCache/llvm-build/scripts/fix-vs.ps1
 
+      - name: Test binary compatibility
```

Comment:
What does this check (aside from its diagnostic output)?  That the llvm-config we built runs on the runner that built it?